### PR TITLE
update to nous-hermes modelfile

### PR DESCRIPTION
**Pull request by technovangelist**
- The nous-hermes model will now accept a system prompt from a model that uses nous-hermes.
- also updated the midjourney-prompter to use a better name.

as per Hugging Face (https://huggingface.co/NousResearch/Nous-Hermes-13b#prompt-format), the prompt template is:

```
Prompt Format
The model follows the Alpaca prompt format:

### Instruction:

### Response:

or

### Instruction:

### Input:

### Response:

```

### CODE REVIEW

File: library/modelfiles/nous-hermes
Comment (mxyng) [inline library/modelfiles/nous-hermes:1]: This isn't quite right. This model takes additional context in the `Input` field, not the `Instruction` field. Instead, it should probably be

```
TEMPLATE """
### Instruction:
{{ .Prompt }}

{{- if and .First .System }}
### Input:
{{ .System }}
{{- end }}

### Response:
"""
```

File: library/modelfiles/nous-hermes [hunk 1]
```diff
@@ -3,5 +3,12 @@ TEMPLATE """
...
 
+{{- if and .First .System }}
+### Input:
+{{ .System }}
+{{- end }}
+
 ### Response:
 """
+
+SYSTEM """You are an AI assistant that follows instruction extremely well. Help as much as you can."""
```
Comment (mxyng) [inline library/modelfiles/nous-hermes:14]: There shouldn't be a default SYSTEM. I suggest this be removed

Comment (technovangelist) [reply library/modelfiles/nous-hermes:14]: Why is nous-hermes different than all the others? llam2, orca, vicuna have a default SYSTEM

Comment (BruceMacD) [reply library/modelfiles/nous-hermes:14]: I think this shows that `SYSTEM` might not be the best wording for this in general. I think it should go in the `Instruction` part of this template:

```
### Instruction:
{{ .System }}
### Input:
{{ .Prompt }}
### Response:
```

source: https://huggingface.co/NousResearch/Nous-Hermes-13b

Comment (mxyng) [reply library/modelfiles/nous-hermes:14]: @BruceMacD you're right, my mistake

**Pull request closed without merging**